docs/ref-migrate-0.13-0.20.md

```diff
@@ -228,9 +228,9 @@ results in the following new logic for the `0.20` version:
 
 ## Command Line Changes
 
-The Command Line interface for Cicero and Ergo has been completely overhauled for consistency. Release `0.20` also features new command lines for Concerto and for the new `markdown-transform` project.
+The Command Line interface for Cicero and Ergo has been completely overhauled for consistency. Release `0.20` also features new command line commands for Concerto and for the new `markdown-transform` project.
```

Comment:
I'm unclear. Command line commands make me think that it's individual commands of the CLIs. Maybe `new command lines interfaces` ?